vxquery-xtest/src/main/java/org/apache/vxquery/xtest/TestRunner.java

```diff
@@ -181,11 +181,13 @@ public TestCaseResult run(final TestCase testCase) {
             res.time = end - start;
         }
         if (opts.showResult) {
```

Comment:
This change may be in master. Please rebase against master. (I just pushed your documentation changes.)